proto/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
This file already exists in `proto/google/api/annotations.proto`. Just change the reference in the proto file of XUD and remove this one .

Reply:
Yeah forgot to delete that.